src/utils/frame.js

```diff
@@ -146,12 +150,19 @@ const displayNameMap = {
 };
 
 function mapDisplayNames(frame, library) {
-  const map = displayNameMap[library];
   const { displayName } = frame;
-  return (map && map[displayName]) || displayName;
+  return (
+    (displayNameMap[library] && displayNameMap[library][displayName]) ||
+    displayName
+  );
 }
 
-export function annotateFrame(frame: Frame) {
+export const annotateFrames = flow(
+  map(annotateFrame),
+  annotateBabelAsyncFrames
+);
```

Comment:
wow, this is a really elegant solution.

- I've never seen `flow` before!
- I like how this *just* solves it by annotating!

Reply:
Thank you! It took some trial and error to realize, "hey wait a minute, grouping already works fine. why change that?"

Reply:
:thumbsup:

Reply:
re: Flow. Careful! I'll do more FP if you encourage me. Not everyone likes it when I do that :P